go.mod

```diff
@@ -1,64 +1,72 @@
 module github.com/Traceableai/terraform-provider-awsapigateway
 
-go 1.18
+go 1.21
 
 require (
-	github.com/aws/aws-sdk-go-v2 v1.18.0
-	github.com/aws/aws-sdk-go-v2/config v1.15.13
-	github.com/aws/aws-sdk-go-v2/credentials v1.12.8
-	github.com/aws/aws-sdk-go-v2/service/apigateway v1.16.11
-	github.com/aws/aws-sdk-go-v2/service/apigatewayv2 v1.13.11
-	github.com/aws/aws-sdk-go-v2/service/sts v1.16.9
-	github.com/golang/mock v1.2.0
-	github.com/google/uuid v1.3.0
-	github.com/hashicorp/terraform-plugin-docs v0.14.1
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.17.0
-	github.com/stretchr/testify v1.7.0
+	github.com/aws/aws-sdk-go-v2 v1.26.1
```

Comment:
bumped every dependency to latest version, github is flagging vulnerabilities on the current main: https://github.com/Traceableai/terraform-provider-awsapigateway/security/dependabot